pkg/plugin/plugin_test.go

```diff
@@ -210,6 +210,62 @@ var _ = Describe("CNI Plugin", func() {
 				testAddDel(conf, false)
 			})
 		})
+		Context("random mac address on container interface", func() {
+			It("should create eth0 on two different namespace with different mac addresses", func() {
```

Comment:
Chances that you will create two matching MAC addresses are low. Would you consider changing this test so it verifies that the MAC address is picked from our local unicast subnet? You can keep the duplicate validation there.